tests/services/DotbitService.test.ts

```diff
@@ -43,7 +43,7 @@ describe('DotbitService', () => {
     })
 
     it('leonx.bit\'s address is 0xc72b6f66017246d6a7f159f5c2bf358188ad9eca', async () => {
-      expect((await dotbitService.addr('leonx.bit', 'BSC'))).toEqual(
+      expect((await dotbitService.addr('leonx.bit', 'bsc'))).toEqual(
```

Comment:
For input value, 'BSC' or 'bsc' are all fine. We can to transform it to the way we need it in our API.

While for output `symbol`, it would be better to stick to 'BSC'.